test/helpers/config.php

```diff
@@ -82,4 +82,4 @@
 });
 
 error_reporting(E_ALL | E_STRICT);
-?>
\ No newline at end of file
+?>
```

Comment:
Sorry, that's been fixed. Let me know when/if you'd like me to squash these commits into one.

Have you considered adding a [.editorconfig](http://editorconfig.org) file? That would prevent these sorts of issues. I can enter another PR with one if that sounds like something you'd like to see.